templates/e-commerce/product-details.amp.html

```diff
@@ -27,7 +27,7 @@
       }
     </script>
   </amp-state>
-  <main id="content" role="main" class="main" [class]="cart.added ? 'main commerce-cart-added' : 'main'">
```

Comment:
Move this to the body since on iOS devices main isn't in the same position as on desktop.